frontend/fingertips-frontend/components/forms/IndicatorSearchForm/index.tsx

```diff
@@ -20,6 +22,33 @@ export const IndicatorSearchForm = ({
       error={indicatorSearchFormState.message !== undefined}
       data-testid="indicator-search-form"
     >
+      <div
+        style={{
+          color: govukBlack,
+          fontFamily: '"nta", Arial, sans-serif',
+          fontSize: '19px',
+          fontWeight: 400,
+          lineHeight: '25px',
+          textAlign: 'left',
+          paddingBottom: '2px',
+          marginBottom: '0px',
+        }}
+      >
+        Search by Subject
+      </div>
+      <div
+        style={{
+          color: govukLightGrey,
+          fontFamily: '"nta", Arial, sans-serif',
+          fontSize: '19px',
+          fontWeight: 400,
+          lineHeight: '25px',
+          textAlign: 'left',
+          marginBottom: '15px',
+        }}
+      >
+        For example smoking, diabetes prevalence, or a specific indicator ID
+      </div>
```

Comment:
Can we use the govuk react `Paragraph` component instead please https://govuk-react.github.io/govuk-react/?path=/docs/paragraph--docs 
This will give us lot of the styling and responsiveness for free. You may still need to tweak it to match the designs.